s2i/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM docker:19.03.2
```

Comment:
Not a blocker, but more of a question. Is there any reason why we're avoiding using a ubi image as the base?

Reply:
The s2i binary is pretty tightly integrated with Docker (see https://github.com/openshift/source-to-image/issues/966). I would love to see a UBI/Podman approach supported by that project, but until then, I think we need to use Docker - and I wasn't aware of a UBI that had actual Docker already installed & didn't necessarily want to install it onto a UBI at action-build time...

Reply:
Sounds good. Possibly something to look into later then.